mcs/class/aot-compiler/Makefile

```diff
@@ -72,6 +73,26 @@ $(csc_SRM_image): $(csc_SRM_dll) $(runtime_dep)
 $(csc_SCI_image): $(csc_SCI_dll) $(runtime_dep)
 	$(Q_AOT) MONO_PATH='$(the_libdir)' > $(LOG_FILE) 2>&1 $(RUNTIME) --aot=bind-to-runtime-version$(profile_arg),outfile=$(csc_SCI_image) --debug $(csc_SCI_dll) || (cat $(LOG_FILE); exit 1)
 
+MKBUNDLE_EXE = $(topdir)/class/lib/$(PROFILE)/mkbundle.exe
```

Comment:
Please don't do it this way, add it to build tools https://github.com/mono/mono/blob/master/mcs/tools/Makefile#L54 instead